docs/developer-guide/shipping-bytecode.md

```diff
@@ -46,9 +46,6 @@ These labels are defined as follows:
 
 - `io.ebpf.section_name`: The section name of the eBPF Program.
 
-- `io.ebpf.kernel_version`: The Kernel version for which this bytecode was compiled
```

Comment:
It was harder to build the images in CI since we aren't yet compiling the bytecode on the GH runners.... Also we weren't even using that arg in bpfd.. From all that it made sense to get rid of it :)